include/grpc/grpc_security.h

```diff
@@ -191,6 +191,12 @@ typedef struct {
      try to get the roots set by grpc_override_ssl_default_roots. Eventually,
      if all these fail, it will try to get the roots from a well-known place on
      disk (in the grpc install directory).
+
+     It is a known issue that when the default root certificates are offered
+     with this parameter, occasionally an excessive amount of memory (~10MB) is
```

Comment:
This is inaccurate and misleading.  The root cert store is ~1MB. But gRPC opens multiple subchannels, each subchannel has its own SSL context and root store. 

I would rephrase as

gRPC has implemented root certificates cache if the underlying OpenSSL library supports it. The gRPC root certificates cache is only applicable on the default root certificates. If user provides his or her own pem_root_certs when creating an SSL credential object, gRPC would not be able to cache it.

Reply:
looks good :) I'll rephrase